### PR TITLE
(#89) college year course

### PR DESCRIPTION
**Pull request by bohraji**

### CODE REVIEW

File: app/Http/Controllers/HR/ApplicantController.php [hunk 1]
```diff
@@ -49,6 +49,11 @@ public function store(ApplicantRequest $request)
             'email' => $validated['email'],
             'phone' => $validated['phone'] ?? null,
             'resume' => $validated['resume'],
+            'college' => $validated['college'] ?? null,
+            'graduation_year' => $validated['graduation_year'] ?? null,
+            'course' => $validated['course'] ?? null,
+            'linkedin' => $validated['linkedin'] ?? null,
```
Comment (nishanth-kd) [inline app/Http/Controllers/HR/ApplicantController.php]: umm whats the significance of this if the second operand is null itself. Should I not just put up `$validated['']`

File: app/Models/HR/Applicant.php [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
 class Applicant extends Model
 {
-    protected $fillable = ['name', 'email', 'phone', 'resume', 'status', 'hr_job_id'];
+    protected $fillable = ['name', 'email', 'phone', 'resume', 'college', 'graduation_year', 'course', 'status', 'linkedin', 'reason_for_eligibility', 'hr_job_id'];
```
Comment (nishanth-kd) [inline app/Models/HR/Applicant.php]: let's move to $gaurded.

File: resources/views/hr/applicant/edit.blade.php [hunk 3]
```diff
@@ -34,6 +39,22 @@
...
+                            @if ($applicant->college)
+                                <div>{{ $applicant->college }}</div>
+                            @else
+                                <div>-</div>
+                            @endif
+                        </div>
+                        <div class="form-group offset-md-1 col-md-5">
+                            <b>Course</b>
+                            @if ($applicant->course)
+                                <div>{{ $applicant->course }}</div>
```
Comment (nishanth-kd) [inline resources/views/hr/applicant/edit.blade.php]: put the div outside ?

File: app/Http/Controllers/HR/ApplicantController.php (hunk 1 above)
Comment (rathorevaibhav) [reply app/Http/Controllers/HR/ApplicantController.php]: Agreed. this makes sense.

**Comment by rathorevaibhav** [comment]
done with the feedback. Merging.

**Pull request merged**